tests/test_conversion.py

```diff
@@ -413,16 +413,10 @@ def test_single_conversion():
     assert ob.SingleField == 0.0
 
     ob.SingleField = 3.402823e38
-    assert ob.SingleField == 3.402823e38
+    assert ob.SingleField == System.Single(3.402823e38)
```

Comment:
When comparing to the `float` literal instead, does this fail with an exception or just return `False`?

Reply:
There's no exception. The reason you have to explicitly cast the literal to `float32` is that `(double)3.402823e38 != (float)3.402823e38`. This was previously hacked around in `Converter` via gross conversion to string representation and back, and I removed that hack.